src/main/java/ezra/Event.java

```diff
@@ -0,0 +1,83 @@
+package ezra;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a task with a specific start and end time.
+ */
+public class Event extends Task {
+
+    protected LocalDateTime start;
+    protected LocalDateTime end;
+    protected String startInput;
+    protected String endInput;
+
+    /**
+     * Constructs an Event object with the specified description, start time, and end time.
+     *
+     * @param description The description of the task.
+     * @param start       The start time of the event in the format "dd/MM/yyyy HHmm".
+     * @param end         The end time of the event in the format "dd/MM/yyyy HHmm".
+     * @throws DateTimeParseException If the start or end time format is invalid.
+     */
+    public Event(String description, String start, String end) throws DateTimeParseException {
+        super(description);
+        this.start = LocalDateTime.parse(start, DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm"));
+        this.end = LocalDateTime.parse(end, DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm"));
+        this.startInput = start;
+        this.endInput = end;
+    }
+
+    /**
+     * Returns a formatted string representation of the Event object to display to the user.
+     *
+     * @return A formatted string including task type, status, description, start time, and end time.
+     */
+    @Override
+    public String toString() {
+        String startString = this.start.format(DateTimeFormatter.ofPattern("dd MMM yyyy hh:mm a"));
+        String endString = this.end.format(DateTimeFormatter.ofPattern("dd MMM yyyy hh:mm a"));
+        return String.format(
+                "[E][%s] %s (from: %s to: %s)",
+                this.getStatusIcon(),
+                this.description,
+                startString,
+                endString
+        );
```

Comment:
I like your formatting of the String.format method here. It is easy to read, and you comply with the 8 space continuation indent standard that many others may miss.